tools/ci/appveyor/appveyor_build.sh

```diff
@@ -26,6 +26,8 @@ else
   MAKE=make
 fi
 
+git config --global --add safe.directory '*'
```

Comment:
Why was this not necessary before?

Reply:
Better late than never: this was due to a Git CVE.
https://github.blog/2022-04-12-git-security-vulnerability-announced/#cve-2022-24765
To mitigate, Git started issuing warnings on Windows (in some cases) if it encounters a new Git repository. Affected users have to register globally all paths leading to Git repositories.
In our CI we don't care, so we can allow everything. It prevents git from exiting with an error status.